renovate.json

```diff
@@ -15,6 +12,12 @@
       "matchManagers": ["gomod"],
       "matchDepTypes": ["indirect"],
       "enabled": true
+    },
+    {
+      "matchFileNames": ["build/**"],
+      "matchManagers": ["gomod"],
+      "matchDepTypes": ["indirect"],
+      "enabled": false
```

Comment:
Why false?

Reply:
To disable it as it is enabled in previous rule.